app/views/kalibro_configurations/_form.html.erb

```diff
@@ -18,6 +18,13 @@
       </div>
     </div>
 
+    <div class="form-row">
+      <div class="field-container checkbox">
+        <label>
+          <%= check_box :attributes, :public %> <%= t('should_be_public') %>
```

Comment:
I think we should not allow the user to hide his KalibroConfigurations. This is a feature just for administrators to remove "test" configurations that pollute the system.

The objective of Mezuro is sharing software metrics usage.